modules/nodes-cluster-overcommit-node-enforcing.adoc

```diff
@@ -18,54 +18,40 @@ If you disable CPU limit enforcement, it is important to understand the impact o
 
 .Prerequisites
 
-. Obtain the label associated with the static `MachineConfigPool` CRD for the type of node you want to configure. Perform one of the following steps:
-
-.. View the machine config pool:
+. Obtain the label associated with the static `MachineConfigPool` CRD for the type of node you want to configure:
```

Comment:
As per OCP guidelines, we need a variation of "Enter this command". For example, 

"Obtain the label associated with the static `MachineConfigPool` CRD for the type of node you want to configure by running the following command:" 

See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting for reference